src/main/java/com/labsynch/labseer/dto/BulkLoadPropertiesDTO.java

```diff
@@ -54,8 +51,8 @@
     
     private String templateName;
     
-    public BulkLoadPropertiesDTO(){
-    	this.dbProperties = propertiesUtilService.getDbProperties();
+    public BulkLoadPropertiesDTO(Collection<SimpleBulkLoadPropertyDTO> dbProperties){
```

Comment:
Nice, I like this. Makes this class slightly more flexible in that it can dynamically consume any set of `SimpleBulkLoadPropertyDTO`, and pushes the reading from configs outside of this DTO class.